sdk/maps/azure-maps-examples/src/main/java/com/azure/maps/examples/RenderSample.java

```diff
@@ -68,46 +71,52 @@ public static void main(String[] args) throws IOException {
-        MapsCommon.print(client.getCopyrightForWorld(ResponseFormat.JSON, null));
+        // System.out.println("Get copyright for world");
+        // MapsCommon.print(client.getCopyrightForWorld(ResponseFormat.JSON, null));
 
-        System.out.println("Get copyright from bounding box");
-        MapsCommon.print(client.getCopyrightFromBoundingBox(ResponseFormat.JSON,
-            new BoundingBox()
-                .setNorthEast(Arrays.asList(52.41064,4.84228))
-                .setSouthWest(Arrays.asList(52.41072,4.84239)),
-            null));
+        // System.out.println("Get copyright from bounding box");
+        // MapsCommon.print(client.getCopyrightFromBoundingBox(ResponseFormat.JSON,
+        //     new BoundingBox()
+        //         .setNorthEast(Arrays.asList(52.41064,4.84228))
+        //         .setSouthWest(Arrays.asList(52.41072,4.84239)),
+        //     null));
 
-        System.out.println("Get map imagery tile");
-        // client.getMapIm
-        // openImageFile(client.getMapImageryTileV21(RasterTileFormat.PNG, MapImageryStyle.SATELLITE, 6, 10, 22));
+        // System.out.println("Get map imagery tile");
+        // // client.getMapIm
+        // // openImageFile(client.getMapImageryTileV21(RasterTileFormat.PNG, MapImageryStyle.SATELLITE, 6, 10, 22));
 
-        /*
-        System.out.println("Get map satelite tile");
-        openImageFile(client.getRenders().getMapStateTilePreview(6, 10, 22, statesetId));
-        */
+        // /*
+        // System.out.println("Get map satelite tile");
+        // openImageFile(client.getRenders().getMapStateTilePreview(6, 10, 22, statesetId));
+        // */
 
-        System.out.println("Get map static image");
-        openImageFile(client.getMapStaticImage(RasterTileFormat.PNG, StaticMapLayer.HYBRID,
-            MapImageStyle.MAIN, 14, null, Arrays.asList(-122.338257,47.572299,-122.297745,47.609346),
-            null, null, null, null, null, null));
+        // System.out.println("Get map static image");
+        // openImageFile(client.getMapStaticImage(RasterTileFormat.PNG, StaticMapLayer.HYBRID,
+        //     MapImageStyle.MAIN, 14, null, Arrays.asList(-122.338257,47.572299,-122.297745,47.609346),
+        //     null, null, null, null, null, null));
 
         System.out.println("Get map tile");
-        openImageFile(client.getMapTileV2(TilesetID.MICROSOFT_BASE_HYBRID_ROAD,
+        // openImageFile(client.getMapTileV2(TilesetID.MICROSOFT_BASE_HYBRID_ROAD,
+        //     new TileIndex().setX(10).setY(22).setZ(6), null, MapTileSize.SIZE512,
+        //     null, null));
+        InputStream result = client.getMapTileV2(TilesetID.MICROSOFT_BASE_HYBRID_ROAD,
             new TileIndex().setX(10).setY(22).setZ(6), null, MapTileSize.SIZE512,
-            null, null));
+            null, null);
+        String toprint = CharStreams.toString(new InputStreamReader(
+                result, Charsets.UTF_8));
+        System.out.println(toprint);
```

Comment:
No System.out.println() and I wouldn't print this at all. Isn't the result binary?